### DISCUSSION TITLE
Generating SEB config file outside SEB configuration tool.

### DISCUSSION THREAD

**Discussion by ghost** [SafeExamBrowser/seb-mac#322] [General]
I encountered challenges while trying to generate SEB config files within the exam app that I am currently developing. I referred to the documentation available at https://safeexambrowser.org/developer/seb-file-format.html  but faced certain difficulties. To proceed with the SEB config generation, I am in need of further insights and documentation. Specifically, I would greatly appreciate information on the required XML structure and the encryption process encryption requirements.

**Comment by danschlet** [comment]
You can search GitHub issues (also closed ones) and discussions in this repository and the Windows repository. There might be some answers in the SourceForge forums, see our [support page](https://safeexambrowser.org/support_en.html). Also reading [Mac release notes](https://safeexambrowser.org/macosx/mac_release_notes_en.html) can help, as details about settings changes usually are mentioned there. You can also check open source implementations of generating SEB config files in SEB for macOS/iOS (Objective-C), SEB for Windows (C# .NET), SEB Server (Java), Moodle (PHP) or OpenOlat (Java).

In general we can only answer single, specific questions here. Consulting about implementing SEB integrations is only available for [Gold Contributors](https://safeexambrowser.org/alliance/fees.html) or higher to the [SEB Alliance](https://safeexambrowser.org/alliance/benefits.html).
